web/cobrands/bristol/assets.js

```diff
@@ -26,6 +26,50 @@ var options = {
     }
 };
 
+// This is required so that the found/not found actions are fired on category
+// select and pin move rather than just on asset select/not select.
+OpenLayers.Layer.BristolVectorAsset = OpenLayers.Class(OpenLayers.Layer.VectorAsset, {
+    initialize: function(name, options) {
+        OpenLayers.Layer.VectorAsset.prototype.initialize.apply(this, arguments);
+        $(fixmystreet).on('maps:update_pin', this.checkSelected.bind(this));
+        $(fixmystreet).on('report_new:category_change', this.checkSelected.bind(this));
+    },
+
+    CLASS_NAME: 'OpenLayers.Layer.PeterboroughVectorAsset'
+});
+
+var parkOptions = $.extend(true, {}, options, {
+    filter_key: '',
+    wfs_url: 'https://tilma.staging.mysociety.org/mapserver/bristol',
+    wfs_feature: "parks",
+    propertyNames: [ ].push( 'SITE_CODE', 'SITE_NAME'),
+    asset_type: 'area',
+    asset_id_field: 'SITE_CODE',
+    srsName: "EPSG:3857",
+    class: OpenLayers.Layer.BristolVectorAsset,
+    strategy_class: OpenLayers.Strategy.FixMyStreet,
+    select_action: true,
+    actions: {
+        asset_found: function(asset) {
+            fixmystreet.message_controller.asset_found.call(this);
+        },
+        asset_not_found: function(layer, asset) {
+            fixmystreet.message_controller.asset_not_found.call(this);
+        }
+    }
+});
+
+
+
+fixmystreet.assets.add(parkOptions, {
+    asset_group: "Parks Maintenance",
+    asset_item: 'park',
+});
+
+fixmystreet.assets.add(parkOptions, {
+    asset_category: "Path Cleaning",
```

Comment:
So you have to pick a park when you select 'Path cleaning', is that right? Where will this category show (it's not on staging?)

Reply:
It's inside of 'Parks Maintenance' But when another asset is picked that loads its own layer, changing your mind and selecting 'Path Cleaning' after that results in a message telling you to pick a park even though you already have.

Reply:
As per our conversation, I've changed from trying to force it to work with asset_category and gone with individual categories